### ISSUE TITLE
PhoneGap build fails - admob plugin conflict

### ISSUE THREAD

**Issue by blukis** [mauron85/cordova-plugin-background-geolocation#118]
PhoneGap build fails when these two plugins are specified in config.xml:
`	<plugin name="admob" version="5.4.2"/>`
`	<plugin name="cordova-plugin-mauron85-background-geolocation" version="2.1.2"/>`

Removing the admob plugin seems to fix the build issue.  Any way to work around this? 

Failing .zip that fails to build in PhoneGap build is attached.  Complete PG build log output is attached.  Thanks!

[www (4).zip](https://github.com/mauron85/cordova-plugin-background-geolocation/files/369265/www.4.zip)
[pgbuild-error-log.txt](https://github.com/mauron85/cordova-plugin-background-geolocation/files/369270/pgbuild-error-log.txt)

**Comment by mauron85** [comment]
Admob is in conflict with this plugin and you cannot make it work with this plugin.

I've checked admob [plugin.xml](https://github.com/admob-google/admob-cordova/blob/master/plugin.xml). I've found multiple problems:

1. It is including google play services as jar (WRONG).
2. It is overriding `com.google.android.gms.version` in meta attribute

Actually the problem 2. is mentioned in your error log.

```
:processReleaseManifest/project/AndroidManifest.xml:15:9 Error:
        Element meta-data#com.google.android.gms.version at AndroidManifest.xml:15:9 duplicated with element declared at AndroidManifest.xml:13:9
/project/AndroidManifest.xml:0:0 Error:
```
